mods/ra/rules/infantry.yaml

```diff
@@ -506,7 +506,6 @@ THF:
 	Passenger:
 		PipType: Yellow
 	Infiltrates:
-		InfiltrateTypes: Cash
```

Comment:
We don't define Cash as an infiltratable types, so renaming this to `Types` would still have been wrong.